rules/integrations/azure/privilege_escalation_azure_kubernetes_rolebinding_modified.toml

```diff
@@ -0,0 +1,47 @@
+[metadata]
+creation_date = "2021/10/18"
+maturity = "production"
+updated_date = "2021/11/22"
+integration = "azure"
+
+[rule]
+author = ["Austin Songer"]
+description = """
+Identifies the modification of role binding or cluster role bindings. You can assign these roles to Kubernetes subjects 
+(users, groups, or service accounts) with role bindings and cluster role bindings. An adversary who has permissions to create
+bindings and cluster-bindings in the cluster can create a binding to the cluster-admin ClusterRole or to other high privileges
+roles. 
```

Comment:
We still need to adjust this. As stated on https://github.com/elastic/detection-rules/pull/1576#discussion_r732591915, "An adversary who has permissions to create bindings and cluster-bindings in the cluster can create a binding to the cluster-admin ClusterRole or to other high privileges roles. this would be an creation event, not modification"